src/table.d.ts

```diff
@@ -0,0 +1,8 @@
+import "@tanstack/react-table"; //or vue, svelte, solid, qwik, etc.
+
+declare module "@tanstack/react-table" {
```

Comment:
We need to check if this augment works in the parent package as well, but for now I think its fine :+1:

Reply:
yeah there were errors here but after this it's working. let's test in the parent package once we release